rmf_tasks/src/agv/TaskPlanner.cpp

```diff
@@ -614,6 +614,7 @@ class TaskPlanner::Implementation
   }
   
   Assignments complete_solve(
+    rmf_traffic::Time relative_start_time,
```

Comment:
I might be missing something here but it looks like this new variable is not being used anywhere?